USAGE.md

```diff
@@ -148,6 +149,14 @@ Only useful if you're running [LÖVE](https://love2d.org/).
 
 See also: [`bitser.dumpLoveFile`](#dumplovefile).
 
+## handleMetatables
+
+Controls whether bitser will (de)serialize metatables. false by default.
```

Comment:
I think I see your reason for defaulting to false: it is always safe to turn it off (in the sense that dumping with that setting won't cause additional errors compared to with it being on), but on the other hand, it would mean it might silently do the wrong thing, which I think is worse than an unexpected error.

Reply:
Well, this was rather to keep the consistency with the latest release.
If the new version is supposed to be minor, it should be kept false by default. For a new major version, yes "breakage" is possible as long as users are aware (in the release note for instance). As long as there's a way to toggle it, I don't see any issue :)

I'll consider this a "major version change" and set the default to true, then.